docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,55 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
+
+> Only available in Grafana v7.0+
+
+To be able to use Team Sync feature, set `assertion_attribute_groups` option to the attribute name where you store user groups. Then Grafana will use attribute values extracted from SAML assertion to add user into the groups with the same name configured on the External group sync tab.
+
+[Learn more about Team Sync]({{< relref "../enterprise/team-sync.md" >}})
+
+### Configure role sync
+
+> Only available in Grafana v7.0+
+
+Role sync allows to map user roles from identity provider to Grafana. To enable role sync, set `assertion_attribute_role` option to the attribute name where the role information will be extracted from. After that, configure desired attribute values for each of Grafana roles: Editor, Admin and Super Admin. If obtained role doesn't match any of configured values, `Viewer` role will be assigned to the user. Example configuration:
```

Comment:
Need links to the appropriate sections in the Permissions section (for roles) and Configuaration.

Also, what is "Super Admin"? I do not recall this mentioned anywhere in our current documentation.

Reply:
This is a Grafana Admin also called Super Admin. That mentioned at the [permissions overview](https://grafana.com/docs/grafana/latest/permissions/overview/#grafana-admin) page. Probably, we can change it to `Grafana Admin` if it's more clear.